src/message_util.c

```diff
@@ -0,0 +1,65 @@
+#include <nitro.h>
+#include <string.h>
+
+#include "message.h"
+#include "strbuf.h"
+
+#include "message.h"
+#include "message_util.h"
+#include "string_template.h"
+#include "strbuf.h"
+#include "constants/heap.h"
+#include "constants/message_banks.h"
+#include "constants/narc.h"
+
+Strbuf* MessageUtil_ExpandedStrbuf (StringTemplate *template, MessageLoader *loader, u32 entryID, u32 heapID)
+{
+    Strbuf *ret = NULL;
+    Strbuf *buffer = Strbuf_Init(EXPANDED_STRING_SIZE, HEAP_ID_SYSTEM);
```

Comment:
[nitpick] I'd prefer that this just be called `buf`, for standardization. Or `str`, but preferably `buf`.